src/syscheckd/registry/registry.h

```diff
@@ -67,9 +67,9 @@ cJSON *fim_registry_key_attributes_json(const fim_registry_key *data, const regi
  * @param diff A string holding the difference between the original and new value of the registry.
  * @return A cJSON object holding the generated event, NULL on error.
  */
-cJSON *fim_registry_event(const fim_entry *new,
+cJSON *fim_registry_event(const fim_entry *new_entry,
```

Comment:
This variable has been renamed, but its doxygen description still lists it as `new`